### PR TITLE
[DO NOT MERGE] mdd-263382

### PR DESCRIPTION
**Pull request by mddifilippo89**
initial pull request: linting pending.

### CODE REVIEW

File: src/app/geo-map/geo-map-routes-data.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// tslint:disable:object-literal-sort-keys
+
+export const geomaproutesdata = {
```
Comment (HUSSAR-mtrela) [inline src/app/geo-map/geo-map-routes-data.ts]: please always use camel case for variable names, e.g. GeoMapRoutesData

File: src/app/geo-map/map-binding-data-csv/map-binding-data-csv.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (HUSSAR-mtrela) [inline src/app/geo-map/map-binding-data-csv/map-binding-data-csv.component.spec.ts]: do we really need .spec.ts file for our samples? Is this a new requirement?

File: src/app/geo-map/geo-map-routes-data.ts (hunk 1 above)
Comment (mddifilippo89) [reply src/app/geo-map/geo-map-routes-data.ts]: will be fixed in my latest commit.

File: src/app/geo-map/map-binding-data-csv/map-binding-data-csv.component.spec.ts (hunk 2 above)
Comment (mddifilippo89) [reply src/app/geo-map/map-binding-data-csv/map-binding-data-csv.component.spec.ts]: all will be removed in my latest commit

**Comment by zdrawku** [comment]
@mddifilippo89 this PR should target `master` branch, as it is for the migration to ng 8

**Comment by HUSSAR-mtrela** [comment]
@mddifilippo89 please fix lint errors

**Comment by HUSSAR-mtrela** [comment]
This PR is obsolete because other PR added GeoMap samples

**Pull request closed without merging**